src/corePDEs/problems/Albany_PopulateMesh.cpp

```diff
@@ -82,7 +82,7 @@ void PopulateMesh::buildProblem (Teuchos::ArrayRCP<Teuchos::RCP<MeshSpecsStruct>
       const int numSideVecDim   = -1;
 
       dl->side_layouts[ss_name] = Teuchos::rcp(new Layouts(numSideVertices,numSideNodes,numSideQPs,
-                                                           numSideDim,numCellDim,numCellSides,numSideVecDim,ss_name));
```

Comment:
This parameter was never used, so I decided to change the signature of the method. It used to be used when we had layouts of the form <Cell,Side,...>, but after Max's work, we just have <Side,...>, so this is now pointless.